.travis.yml

```diff
@@ -1,5 +1,14 @@
 language: node_js
 before_install: "! grep PLEASE_FILL_IN_HERE README.md"
+before_script:
+    - psql -c 'create database travis_ci_test;' -U postgres
 node_js:
-  - '6'
   - '7'
+  - '8'
+env:
+  # HACK(tim): db urls for sqlite are broken, setting to '' will use the
+  # default settings for the test to run (sqlite) and still make an explit
+  # build for that case.
+  # https://github.com/tgriesser/knex/issues/2038
+  - TEST_DATABASE_URL=''
+  - TEST_DATABASE_URL='postgres://localhost/travis_ci_test'
```

Comment:
It runs does the whole combinations thing... i.e. a build for each set of env vars (you can have multiple per line) for each environment (node.js 7 & 8).  See travis docs [here](https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item)